test/framework/integration_framework/fake_peer/fake_peer.cpp

```diff
@@ -212,6 +212,8 @@ FakePeer::getProposalStorage() {
 std::unique_ptr<iroha::network::ServerRunner> FakePeer::run() {
   ensureInitialized();
   // start instance
+  using namespace std::chrono_literals;
+  std::this_thread::sleep_for(500ms);
```

Comment:
Seems like a hack. Plz insert TODO to wait untill server stop and all handlers will be joined

Reply:
removed